### PR TITLE
- Adapt module so it can be called during first stage of installation

### PR DESCRIPTION
**Pull request by jsuchome**
(which is the only stage in SLES12)

### CODE REVIEW

File: package/yast2-crowbar.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Oct  7 13:52:52 CEST 2015 - jsuchome@suse.com
+
+- Adapt module so it can be called during first stage of installation
+  (which is the only stage in SLES12)
+- 3.1.8
```
Comment (mchf) [inline package/yast2-crowbar.changes]: I miss bug / fate reference. If none, I'd postpone it for RC2

**Comment by mchf** [comment]
Was it tested at least manually?

File: package/yast2-crowbar.changes (hunk 1 above)
Comment (jsuchome) [inline package/yast2-crowbar.changes]: We do not need it in SLES itself, we ship the module separately

**Comment by jsuchome** [comment]
@mchf yes, it was tested, see bug 949185 (but that bug is for a different part of problem, that's why it is not mentioned here)

**Comment by mchf** [comment]
ok. LGTM

**Pull request merged**